### DISCUSSION TITLE
Why are the outputs in this repo?

### DISCUSSION THREAD

**Discussion by mgred** [sqids/sqids-javascript#18] [Q&A]
I'm wondering, since I think this quite uncommon to publish the outputs (ESM, CJS, UMD) in the source repository :thinking:

**Comment by 4kimov** [comment]
I'm not up to date on latest js packaging techniques, but this mirrors what @niieani does with [hashids](https://github.com/niieani/hashids.js). From my understanding it's just diff ways this lib could be imported, you can see the outputs are referenced in `package.json`

**Comment by niieani** [comment]
The outputs shouldn't be in main, only in the git tags of released versions. I will update it when I revamp the release system.

**Reply by mgred** [reply]
What do you mean by _only in the git tags..._? I think these outputs should never appear in git, only in the resulting package.
